python/setup.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+from setuptools import setup
+
+exec(compile(open("pyspark/pyspark_version.py").read(), 
+   "pyspark/pyspark_version.py", 'exec'))
+VERSION = __version__
+
+setup(name='pyspark',
+    version=VERSION,
+    description='Apache Spark Python API',
+    author='Spark Developers',
+    author_email='dev@spark.apache.org',
+    url='https://github.com/apache/spark/tree/master/python',
+    packages=['pyspark', 'pyspark.mllib', 'pyspark.ml', 'pyspark.sql', 'pyspark.streaming'],
+    install_requires=['numpy>=1.7', 'py4j==0.8.2.1', 'pandas'],
```

Comment:
An alternative which will work for both of you would be to have an extra_requires field which has extra requirements just like ipython does for its notebook.

So there could be something like:

    extra_requires={
        "sql": ["numpy>=1.7", "pandas"]
    }

That way you can install pyspark with `pip install pyspark` and pyspark for spark sql with `pip install pyspark[sql]`.